docs/architecture/adr-003-consumer-expiration

```diff
@@ -0,0 +1,59 @@
+# ADR 004: Consumer expiration
+
+# ADR 004: Consumer chain halts if the VSC packet is no longer received.
+
+## Changelog
+
+- 2023-02-06: Initial draft
+
+## Status
+
+Proposed
+
+## Context
+
+Currently a consumer chain can’t live if the IBC channel is closed or if the channel freezes (which happens if there’s no IBC activity during 2/3 of the provider unbonding period time).
+
+Unbound tokens on the provider chain must ensure the consumer chain have sent a VSC matured Packet to approve the unbonding.
+
+We believe this could be simplified.
+
+## Decision
+
+Let’s add a `max_time_without_vsc_packet` parameter inside the consumer Params and ensure the chain halts in case no validator set updates were received after that time.
+
+This time must be:
+
+- Shorter than 1/2 of the provider unbonding period.
```

Comment:
why does it have to be less than 1/2?

Reply:
To ensure someone don't have the time to unbond on the provider chain and harm the consumer chain then, if there is no VSC package.